include/votca/tools/unitconverter.h

```diff
@@ -88,8 +88,8 @@ enum MolarForceUnit {
 class UnitConverter {
  private:
   /// All distances with respect to Ang
-  constexpr double getDistanceValue_(
-      const DistanceUnit& enum_type) const noexcept {
+  constexpr double getDistanceValue_(const DistanceUnit& enum_type) const
```

Comment:
It seems you are using the wrong `clang-format` version, please use clang-10.

Reply:
we should put a version requirement on the clang-format then